src/backend.rs

```diff
@@ -97,6 +97,6 @@ impl MySqlBackend {
     }
 
     pub fn replace(&mut self, table: &str, vals: Vec<Value>) {
-        self.do_insert(table, vals, true);
+            self.do_insert(table, vals, true);
```

Comment:
spurious indentation; either remove or run `cargo fmt` to auto-format your Rust code.

Reply:
fixed!